terminus-ui/scss/helpers/_typography.scss

```diff
@@ -219,6 +219,7 @@ $typography-body-levels: (
 
   // FIXEDWIDTH
   @if $format == 'fixedwidth' {
+    $typeface: monospace;
```

Comment:
Unused variable

Reply:
ah! I forgot to take this out. I was trying to work around this bad practice:
`@included typography(fixedwidth)` returns
```scss
          .format-test {
            font-family: Roboto, Helvetica Neue, Helvetica, Arial, sans-serif;
            font-family: Lucida Console, Monaco, DejaVu Sans Mono, monospace;
          }
```

Reply:
Ah great point. We can definitely improve our output there. I'll make a ticket 👍